### ISSUE TITLE
Switching themes broken?

### ISSUE THREAD

**Issue by janbrouwer** [thomthom/SKUI#98]
I can't get theme-switching to work unless I change the following file:
https://github.com/thomthom/SKUI/blob/master/src/SKUI/html/window.html

Line 8, from:
<link rel="stylesheet" type="text/css" media="screen" href="../css/core.css" />

to:
<link id="SKUI_CSS_CORE" rel="stylesheet" type="text/css" media="screen" href="../css/core.css" />

Could be that I misunderstood something, or maybe the id accidentally got lost somewhere along the way :-)

**Comment by thomthom** [comment]
Do you try to switch theme after the window has been created?
I need to dig into my examples, but I think one had to set this up in the constructor.

**Comment by janbrouwer** [comment]
I believe I tried both, but currenty I'm setting a personal theme directly.
I can only find one reference to this id in the entire codebase, only in ui.window.js:
line 91: $('#SKUI_CSS_CORE').after( $theme );
where the id from the base-css-stylesheet is needed to set the theme-css-stylesheet.
So I figured it needed to be set in the window.html file, and there I found it had been there before...

**Comment by thomthom** [comment]
Just an FYI - I've been swamped with work and not been able to keep on top of my inbox. I'm going on vacation next week so there will be another week of inactivity. 
Wanted to let you know I'm not deliberately ignoring all these issues - I just need to get caught up.

**Comment by janbrouwer** [comment]
Thanks for the update, no problem! :-)

**Comment by janbrouwer** [comment]
I don't think you already had the chance to look into this, right? No hurry, just checkin'.. ;-)

**Comment by thomthom** [comment]
I'm terrible sorry for being completely absent. I've been so swamped this year so far. :(

**Comment by janbrouwer** [comment]
No problem, I can imagine ;-)
Op 19 jul. 2015 11:53 schreef "Thomas Thomassen" <notifications@github.com>:

**Comment by thomthom** [comment]
It's not forgotten though. I've had some new ideas to this lib that i want to
get done.



Sent using CloudMagic [https://cloudmagic.com/k/d/mailapp?ct=pa&cv=7.0.34&pv=5.0.2] On Sun, Jul 19, 2015 at 5:50 PM, Jan Brouwer < notifications@github.com [notifications@github.com] > wrote:
No problem, I can imagine ;-)
Op 19 jul. 2015 11:53 schreef "Thomas Thomassen" <notifications@github.com>:

> I'm terrible sorry for being completely absent. I've been so swamped this
> year so far. :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thomthom/SKUI/issues/98#issuecomment-122644074>.
>
—
Reply to this email directly or view it on GitHub
[https://github.com/thomthom/SKUI/issues/98#issuecomment-122673414] .[https://github.com/notifications/beacon/AALvoiZ6ub6cMDBiCz-j4gALY2tfcB9Xks5oe77LgaJpZM4Dr7AM.gif]
